### PR TITLE
Open PRs queue stats

### PR DESCRIPTION
**Pull request by Sgitario**
Fix https://github.com/quarkus-qe/gh-activity/issues/3

### CODE REVIEW

File: src/main/java/io/quarkus/activity/GitHubOpenPrQueueService.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    private volatile LocalDateTime updated;
+
+    @ConfigProperty(name = "activity.quarkus-qe.organization", defaultValue = "quarkus-qe")
+    String quarkusQeOrganization;
+
+    @ConfigProperty(name = "activity.start", defaultValue = "2020-01-01")
+    LocalDate statsStart;
+
+    @Scheduled(every = "6H")
```
[the remark this answers is not in the document]
Comment (Sgitario) [reply src/main/java/io/quarkus/activity/GitHubOpenPrQueueService.java:32]: Changed

File: src/main/java/io/quarkus/activity/GitHubOpenPrQueueService.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+public class GitHubOpenPrQueueService {
+
+    @Inject
+    GitHubService gitHubService;
+
+    private volatile OpenPullRequestsQueueByRepositories openPrQueueInOrganization;
+
+    private volatile LocalDateTime updated;
+
+    @ConfigProperty(name = "activity.quarkus-qe.organization", defaultValue = "quarkus-qe")
```
[the remark this answers is not in the document]
Comment (Sgitario) [reply src/main/java/io/quarkus/activity/GitHubOpenPrQueueService.java:26]: Changed

File: src/main/java/io/quarkus/activity/GitHubOpenPrQueueService.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    String quarkusQeOrganization;
+
+    @ConfigProperty(name = "activity.start", defaultValue = "2020-01-01")
```
[the remark this answers is not in the document]
Comment (Sgitario) [reply src/main/java/io/quarkus/activity/GitHubOpenPrQueueService.java:29]: Thanks for spotting. Removed.

File: src/main/java/io/quarkus/activity/GitHubOpenPrQueueService.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    LocalDate statsStart;
+
+    @Scheduled(every = "6H")
+    public void updateOpenPrQueueInOrganization() throws IOException {
+        openPrQueueInOrganization = buildOpenPrQueueInOrganization();
+    }
+
+    public LocalDateTime getUpdated() {
```
[the remark this answers is not in the document]
Comment (Sgitario) [reply src/main/java/io/quarkus/activity/GitHubOpenPrQueueService.java:37]: Changed

File: src/main/java/io/quarkus/activity/ActivityResource.java [hunk 5]
```diff
@@ -50,4 +56,15 @@ public TemplateInstance getMonthlyStats() throws IOException {
...
     }
+
+    @GET
+    @Path("/open-pr-queue")
+    @Produces(MediaType.TEXT_HTML)
+    public TemplateInstance getOpenPrsQueue() throws IOException {
+        return openPrQueue.data(
+                "logins", gitHubService.getLogins(),
+                "organization", gitHubOpenPrQueueService.getOpenPrQueueInOrganization(),
+                "updated", gitHubActivitiesService.getUpdated()
```
[the remark this answers is not in the document]
Comment (Sgitario) [reply src/main/java/io/quarkus/activity/ActivityResource.java:67]: Changed

**Comment by Sgitario** [comment]
@rsvoboda I think I've already addressed all your comments and also I've changed the table to also output the title and some other enhancements. Let me know what you think about them.

File: src/main/resources/templates/openPrQueue.html [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+{#body}
+<div class="ui main container">
+    <div class="ui icon message">
+        <i class="small sync icon"></i>
+        <div class="content">
+            <p>Updated {inject:prettyTime.format(organization.updated)} (refreshed every 10 minutes).</p>
+        </div>
+    </div>
+    {#for entry in organization.repositories}
+    <table class="ui celled striped table">
```
Comment (rsvoboda) [inline src/main/resources/templates/openPrQueue.html:13]: Let's do this to be in synch with Monthly stats
```html
    <h2>{entry.value.size} open Pull Requests in {entry.key}</h2>
    <table class="ui celled striped table">
        <thead>
        <tr>
            <th>Created At</th>
            <th>Title</th>
            <th>Author</th>
            <th>Reviewers</th>
        </tr>
        </thead>
```

**Review by rsvoboda — CHANGES_REQUESTED** [review]

File: src/main/resources/templates/openPrQueue.html (hunk 6 above)
Comment (Sgitario) [reply src/main/resources/templates/openPrQueue.html:14]: Changed.

**Review by rsvoboda — APPROVED** [review]

**Pull request merged**